vrana/adminer/2019-01-20.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Adminer script versions from 4.3.1 up to 4.6.2 contains file disclosure vulnerability
+link:      https://sansec.io/research/adminer-4.6.2-file-disclosure-vulnerability
+reference: composer://vrana/adminer
+cve:       ~
+branches:
+   master:
+       time:     2019-01-20 00:00:00
+       versions: ['>=4.3.1', '<4.6.3']
```

Comment:
The post you link does not say that versions before 4.3.1 are unaffected, but that the researcher stopped testing them. I suggest removing the lower bound here.

Reply:
Fixed, thaks.